scripts/update_deps.py

```diff
@@ -668,7 +671,7 @@ def main():
         '--api',
         dest='api',
         default='vulkan',
-        choices=['vulkan', 'vulkansc'],
```

Comment:
Does this affect sc at all? Or does sc do it’s own modifications making this redundant?

Reply:
This is taken straight from VVL. If there is an issue it would have to be fixed in vvl as well. And I use VVL as the baseline for correctness.

Reply:
Works for me!